docs/config.rst

```diff
@@ -258,13 +258,19 @@ For example::
     "source1": {
       "url": "mbtiles://source1.mbtiles",
       "type": "vector"
+    },
+    "terrain": {
+      "url": "mbtiles://terrain.mbtiles",
+      "encoding": "mapbox"
```

Comment:
we are going to find a good spot to put this documentation since it doesn't just apply to mbtiles. maybe a new section that notes this and the information right below. ir might be good to note it only works with pmtiles or mbtiles

Reply:
I would like the second approach, in general the tileserver handles "data" even if we add some special terrain handling here, I think there is no need to show that in an own section. (This may change, if there will be more features in this section ;-)).

Reply:
I miss understood your comment (I read it as a change of the code regarding configuration). 

I updated the documentation with a dedicated section nested in "data".

Reply:
Thanks it looks better now